GameOfRunes/GameOfRunes/Types/PowerUpType.swift

```diff
@@ -99,6 +62,31 @@ enum PowerUpType: String, CaseIterable {
         }
     }
     
+    func activate(at position: CGPoint, with size: CGSize?, gameEngine: GameEngine) {
+        var powerUpSize = size
+        
+        switch self {
```

Comment:
Think we will need to use `.activationType` for the switch case instead

Reply:
Erm `self` is correct i think. `.activationType` is used in`preparePowerUp`